src/stream/src/from_proto/mview.rs

```diff
@@ -38,13 +38,15 @@ impl ExecutorBuilder for MaterializeExecutorBuilder {
             .map(OrderPair::from_prost)
             .collect();
         let column_ids = node
-            .column_ids
+            .get_table()?
+            .get_columns()
             .iter()
-            .map(|id| ColumnId::from(*id))
+            .map(|t| ColumnId::from(t.column_desc.as_ref().unwrap().column_id))
             .collect();
```

Comment:
Suggest construct StateTable in from_proto use `StateTable::from_table_catalog`, so that we can avoid passing column ids stuff all around.